packages/typespec-ts/test/modularUnit/clientContext.spec.ts

```diff
@@ -2,7 +2,8 @@ import assert from "assert";
 import { emitModularClientContextFromTypeSpec } from "../util/emitUtil.js";
 import { assertEqualContent } from "../util/testUtil.js";
 
-describe("modular client context type", () => {
+// TODO issue tracked https://github.com/Azure/typespec-azure/issues/1598
+describe.skip("modular client context type", () => {
```

Comment:
i notice there are serveral tcgc issues impacted us, do we need to have a followup issue in our side for integration effort?

Reply:
issue tracked https://github.com/Azure/autorest.typescript/issues/2862